### DISCUSSION TITLE
Provide Tailwind-less version of the Tina Cloud Starter

### DISCUSSION THREAD

**Discussion by perkinsjr** [tinacms/tinacms#2514] [Ideas]
We should provide a unopinated starter that contains CSS styling. So that users can choose their own but still have a lot of the project stood up:

Our current tailwind-starter would be a perfect to remove all the Tailwind and provide the user with the content model etc.

**Comment by mitchmac** [comment]
Is the [barebones starter](https://github.com/tinacms/tina-barebones-starter) too minimalist to help for this case @perkinsjr?

**Comment by perkinsjr** [comment]
Yes, our barebones starter doesn't offer all the schema and pages that the cloud-starter does. We want to be able to offer something that has schema work that someone could style using CSS. The barebones is too minimal for this use case, someone might want to use straight CSS and feel spending the time pulling apart out tina-cloud-starter not worth the hassle

**Comment by scottgallant** [comment]
Good idea. Should we rename this discussion from "Provide a CSS only starter" to something like "Provide Tailwind-less version of the Tina Cloud Starter"?  Or am I misunderstanding?

If I'm understanding you correctly, a simple "classless" CSS framework might be nice for this (a framework that only targets semantic HTML elements like [Marx](https://codepen.io/mblode/pen/JdYbJj)). That way, it could be easy-on-the-eyes with minimal styling that's easy to rip out.

**Reply by perkinsjr** [reply]
Renamed! Yup something like Marx may be easiest for us to provide and easy for someone to tear out.

**Comment by amp9020** [comment]
Any progress or timeline for this?=)
